### PR TITLE
fix typos in several files

### PR DESCRIPTION
**Pull request by allencloud**
fix typos in several files

### CODE REVIEW

File: daemon/restart.go [hunk 1]
```diff
@@ -8,7 +8,7 @@ import (
 
 // ContainerRestart stops and starts a container. It attempts to
 // gracefully stop the container within the given timeout, forcefully
-// stopping it if the timeout is exceeded. If given a negative
```
Comment (calavera) [inline daemon/restart.go]: this is not a typo.

Comment (thaJeztah) [inline daemon/restart.go]: agreed

**Comment by thaJeztah** [comment]
thanks! LGTM after @calavera's comment is addressed

**Comment by allencloud** [comment]
@thaJeztah Thanks. I have already fixed that.

**Comment by thaJeztah** [comment]
thanks! LGTM

ping @vdemeester for the merge

**Comment by vdemeester** [comment]
LGTM 🐵

**Pull request merged**